config/routes.rb

```diff
@@ -31,4 +32,6 @@
 
   constraints(RouteConstraints::AccessYourTeachingQualificationsConstraint.new) { draw(:aytq) }
   constraints(RouteConstraints::CheckRecordsConstraint.new) { draw(:check_records) }
+
+  root to: redirect("/qualifications/sign-in")
```

Comment:
This likely an issue if navigating to the root of the Check service. I wonder if defining different `root` in each of the `aytq` and `check_records` route files will do the job?

Reply:
(https://trello.com/c/ylTD364t/18-add-dev-setup-to-serve-up-check-and-aytq-from-different-hostnames)